sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -103,3 +109,28 @@ def get_api_version(kwargs, default):
         versions = '\n'.join(_SUPPORTED_API_VERSIONS)
         raise ValueError("Unsupported API version '{}'. Please select from:\n{}".format(api_version, versions))
     return api_version or default
+
+
+def serialize_blob_tags_header(tags=None):
+    # type: (Optional[Dict[str, str]]) -> str
+    components = list()
+    if tags:
+        for key, value in tags.items():
+            components.append(quote(key, safe='.-'))
+            components.append('=')
+            components.append(quote(value, safe='.-'))
+            components.append('&')
+
+    if components:
+        del components[-1]
+
+    return ''.join(components)
```

Comment:
If you use `'&'.join(components)` then you wont have to delete it off the end in the line above :)

Reply:
currently the component list is like ["key", "=", "value", "&", "key2", "=", "value2", "&" ], so I "&" wasn't used to concatenate those items. While we can make it ["key=value","key2=value2"] and join using "&"